packages/server/lib/errors.js

```diff
@@ -965,6 +965,14 @@ const getMsgByType = function (type, arg1 = {}, arg2, arg3) {
 
         ${chalk.yellow(arg2)}
       `
+    case 'CT_NO_DEV_START_EVENT':
+      return stripIndent`\
```

Comment:
nit: Can we not include the `\` in the message:

```
To run component-testing, cypress needs the \`dev-server:start\` event. 
````

The `\` look a bit weird.

Reply:
The backslash are not in the command message. They are only necessary for the backticks to be displayed in the final error.
My description was confusing.

(I did a poor Copy-Paste)